lobby/src/main/java/games/strategy/engine/lobby/server/db/BannedMacDao.java

```diff
@@ -17,7 +17,7 @@
    * if it already exists.
    *
    * @param bannedUser The user whose MAC will be banned.
-   * @param banTill The instant at which the ban will expire or {@ode null} to ban the MAC forever.
+   * @param banTill The instant at which the ban will expire or {@code null} to ban the MAC forever.
```

Comment:
IntelliJ is really far more polished than eclipse (otherwise the money you'd have to pay wouldn't be worth it).
Luckily I don't have to pay for it at all and can still have the ultimate edition.